test/Domain/LeanCode.DomainModels.MassTransitRelay.Tests/ScopedFiltersTests.cs

```diff
@@ -249,4 +232,6 @@ public void SetConsumer(Guid messageId, Guid value)
             return Data.TryGetValue($"{messageId}_consumer", out var value) ? value : null;
         }
     }
+
+    public void Dispose() => serviceProvider.Dispose();
```

Comment:
It's weird how this doesn't do the same thing as `DisposeAsync`.

Reply:
Well, that xUnit's `IAsyncLifetime` not me :stuck_out_tongue:

Reply:
Ah, and `IAsyncLifetime` is not `IAsyncDisposable`, right :confused: